### ISSUE TITLE
Docs: resolution order of `parserOptions.project`

### ISSUE THREAD

**Issue by jjangga0214** [typescript-eslint/typescript-eslint#5617]
### Before You File a Documentation Request Please Confirm You Have Done The Following...

- [X] I have looked for existing [open or closed documentation requests](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aissue+label%3Adocumentation) that match my proposal.
- [X] I have [read the FAQ](https://typescript-eslint.io/docs/linting/troubleshooting) and my problem is not listed.

### Suggested Changes

Hi!
For example, in this case, which one would be resolved, a or b?

```js
parserOptions: {
  project: ['tsconfig.a.json', 'tsconfig.b.json',], 
}
```

I think this is good to be explicitly mentioned on the docs.

Thanks :)

### Affected URL(s)

https://typescript-eslint.io/docs/linting/typed-linting/monorepos/#one-tsconfigjson-per-package-and-an-optional-one-in-the-root

**Comment by jjangga0214** [comment]
@JoshuaKGoldberg Hi!
This is a request for docs, but I am also curious about the answer.
Which one would be resolved, a or b?
Thanks a lot :)

**Comment by bradzacher** [comment]
It's an array!
The tooling just iterates the array, so a then b

**Comment by kazizi55** [comment]
I would like to work on this issue!

**Comment by JoshuaKGoldberg** [comment]
This was closed by #5682. Yay!
